libgalois/include/katana/PropertyGraph.h

```diff
@@ -659,6 +721,26 @@ class KATANA_EXPORT PropertyGraph {
   /// \return returns the most specific edge entity type for @param edge
   EntityTypeID GetTypeOfEdge(Edge edge) const {
     return edge_entity_type_id_[edge];
+    
+  size_t GetNodeTypeSetIDArraySize() const { return node_type_set_id_.size(); }
+
+  size_t GetEdgeTypeSetIDArraySize() const { return edge_type_set_id_.size(); }
```

Comment:
Why are these two required? They should always be the number of nodes and edges in the graph respectively.

Reply:
Those functions are there to validate that assumption.

Reply:
That should be validated when it is constructed. It shouldn't need to be exposed in the API.